Security Analytics/SIEM-examples/README.md

```diff
@@ -2,4 +2,5 @@
 
 SIEM examples, such as scripts and APIs:
 
-[**Detections API Postman collection**](Detections-API/Kibana.postman_collection.v2.json)
\ No newline at end of file
+[**Detections API Postman collection**](Detections-API/Kibana.postman_collection.v2.json)
+[**Alternative Ingest Pipeline for Packetbeat adding ASN details**](Packetbeat/geoip-info.json) An alternative to the pipeline from [docs](https://www.elastic.co/guide/en/beats/packetbeat/7.6/packetbeat-geoip.html#packetbeat-configuring-geoip)
```

Comment:
Link should point to current:

https://www.elastic.co/guide/en/beats/packetbeat/current/packetbeat-geoip.html#packetbeat-configuring-geoip

Reply:
fixed in followup commit. thx

Reply:
Great - thanks.